src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -4570,6 +4570,32 @@ static void partGroupStart(XmlWriter& xml, int number, const BracketItem* const
 //---------------------------------------------------------
 //   findMetronome
 //---------------------------------------------------------
+static size_t indexOf(const String& src_, const std::wregex& re, size_t from, std::wsmatch* m)
```

Comment:
Should be in String class?

Reply:
I don't like this method, I wouldn't want it used, I added it for compatibility reasons